scripts/dev/license-check.py

```diff
@@ -30,7 +30,4 @@
 
 # Check files
 for base in dirs:
```

Comment:
So are we just not checking on the subdirectories any more?  Maybe it's just a little hard to tell the intent from this small diff.

Reply:
No, I've changed to using Python's glob module differently so it should now hit all subdirectories now.